official-antistasi-community/A3-Antistasi
# "Send Chat Notification When Downed" QOL Parameter

## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:

Added a parameter that will send a notification in chat when you are downed/injured. 

It helps in cases where players are downed and nobody realises.

### Please specify which Issue this PR Resolves.
N/A

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [x] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

The notification does not trigger with ace medical since [fn_handleDamage.sqf](https://github.com/official-antistasi-community/A3-Antistasi/blob/unstable/A3A/addons/core/functions/Revive/fn_handleDamage.sqf#L42-L43) exits early if ace medical is present.

I suppose it'd be possible to make it work with ace if the code was replicated in `fn_initACEUnconsciousHandler.sqf`. Any opinions on that would be appreciated!

Also, if approved localization/translation will need to be added though I'm not entirely sure how that's handled here in Community. I know there will be issues with the message being remoteExec'd after localizing it, not sure if there is a method to handle that yet.

### How can the changes be tested?
Steps:

Set the new "Send Chat Notification When Downed" parameter at the bottom of the "Game Mode" parameters to "Yes".

Proceed to be injured and downed, observe the notification appear in the global chat.

********************************************************
Notes:

I can see how the majority might not use this parameter but a few may appreciate it.
